tests/test_query.py

```diff
@@ -115,3 +117,19 @@ def test_get_latest_versions(
     result = get_latest_versions(url, ids)
 
     assert result == expected
+
+
+@pytest.mark.parametrize("ids", [(["foo", "bar"])])
+def test_get_latest_versions_error(versions_response_error, ids: List[str],) -> None:
+    url = "https://example.com"
+    versions_response_error(url + "/files/versions")
+
+    with pytest.raises(Exception) as e:
```

Comment:
Using a more nuanced exception also allows this part of the test to be a little more meaningful as it will be looking for an expected exception type. Another note you can use the match property to do the checking of the error message instead of doing it below.